dm/worker/server.go

```diff
@@ -648,18 +653,20 @@ func (s *Server) startWorker(cfg *config.SourceConfig) error {
 		subTaskCfgs = append(subTaskCfgs, &subTaskCfg)
 	}
 
-	dctx, dcancel := context.WithTimeout(s.etcdClient.Ctx(), time.Duration(len(subTaskCfgs))*3*time.Second)
-	defer dcancel()
-	minPos, err := getMinPosInAllSubTasks(dctx, subTaskCfgs)
-	if err != nil {
-		return err
-	}
+	if cfg.EnableRelay {
+		dctx, dcancel := context.WithTimeout(s.etcdClient.Ctx(), time.Duration(len(subTaskCfgs))*3*time.Second)
+		defer dcancel()
+		minPos, err := getMinPosInAllSubTasks(dctx, subTaskCfgs)
```

Comment:
`declaration of "err" shadows declaration at line 637` reported by `make check`.

Reply:
fix in 812b25f